lib/atomic_time/constructors/class_methods.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+class AtomicTime
+  module Constructors
+    module ClassMethods
+      def at(secs)
```

Comment:
This is a little unorthodox - internally, I've made `at` the primary constructor since it's the constructor that accepts seconds (the form that data is primarily stored. You'll see below that `new` is overridden to use `at`.